applications/timescaledb-single/chart/templates/tests/test-config.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "{{ .Release.Name }}-tsdb-test"
+  labels:
+    app: {{ template "timescaledb.fullname" . }}
+    chart: {{ template "timescaledb.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    cluster-name: {{ template "clusterName" . }}
+  annotations:
+    argocd.argoproj.io/hook: PostSync
+    argocd.argoproj.io/hook-delete-policy: HookSucceeded
+data:
+  test.sh: |-
+    # List current databases
+    echo "* List current databases"
+    psql -U admin -h {{ template "clusterName" . }} postgres -c "SELECT datname FROM pg_database;"
+
+    # Remove test database if it exists
+    echo "* Remove test db if exists before test"
+    psql -U admin -h {{ template "clusterName" . }} postgres -c "SELECT datname FROM pg_database;"  | grep test_helm_chart &&   psql -U admin -h {{ template "clusterName" . }} postgres -c "drop database test_helm_chart ;"
+
+    # Create test database
+    echo "* Create test database"
```

Comment:
I guess probability is not high, but this can lead to problems if someone uses the same db name. What do you think about making it configurable? Or at least document it somewhere in the readme

Reply:
To be able to finish the task we decided to use the fixed name "test_helm_chart". We though this name would be away of any production database name or atlease far away names of fiware platform

Reply:
Ok